draft-ietf-cbor-7049bis.md

```diff
@@ -1998,6 +1998,9 @@ rules in {{-iana}}. IANA has also assigned a new MIME media type and
 an associated Constrained Application Protocol (CoAP) Content-Format
 entry.
 
+\[To be removed by RFC editor:]
```

Comment:
IANA is going to prefer that this stays in the document so their instructions do not vanish during Auth8

Reply:
Maybe, but by then they will have carried out those instructions.
(They are pretty simple/obvious instructions anyway.)